src/games/strategy/triplea/ai/proAI/ProAI.java

```diff
@@ -90,6 +92,12 @@ public ProAI(final String name, final String type) {
     storedPurchaseTerritories = null;
     storedPoliticalActions = null;
     storedStrafingTerritories = new ArrayList<Territory>();
+    initializeBattleCalculator();
+  }
+
+  public void initializeBattleCalculator() {
+    final IOddsCalculator calc = new ConcurrentOddsCalculator("ProAI");
+    ProBattleUtils.setOddsCalculator(calc);
```

Comment:
Good point. I hadn't considered that you might want some of each AI in a game. I'll need to rework this some.